src/plugins/guided_onboarding/public/components/quit_guide_modal.tsx

```diff
@@ -27,32 +36,52 @@ export const QuitGuideModal = ({ closeModal, currentGuide }: QuitGuideModalProps
   };
 
   return (
-    <EuiConfirmModal
-      maxWidth={448}
-      title={i18n.translate('guidedOnboarding.quitGuideModal.modalTitle', {
-        defaultMessage: 'Quit this guide?',
-      })}
-      onCancel={closeModal}
-      onConfirm={deleteGuide}
-      cancelButtonText={i18n.translate('guidedOnboarding.quitGuideModal.cancelButtonLabel', {
-        defaultMessage: 'Cancel',
-      })}
-      confirmButtonText={i18n.translate('guidedOnboarding.quitGuideModal.quitButtonLabel', {
-        defaultMessage: 'Quit guide',
-      })}
+    <EuiModal
```

Comment:
I changed this to `EuiModal` in order to apply a custom `data-test-subj` to the confirmation button.